local/local.go

```diff
@@ -482,6 +482,10 @@ func (i *Image) Delete() error {
 	return err
 }
 
+func (i *Image) ManifestSize() (int64, error) {
+	return 0, nil
```

Comment:
I'm not sure if manifest size is important for `local` but it does look like a manifest is generated elsewhere in the file, which you could potentially extract to a shared function and do a `len` on the return: https://github.com/buildpacks/imgutil/blob/ae3e2fcd82781e7c6aba7bc25f3e111467e44e11/local/local.go#L432-L438

Reply:
This is interesting.
According to https://github.com/buildpacks/spec/issues/169:
```This field will not apply to the -daemon case, b/c no manifest has been generated.```
Let me know if you still think we should change the `ManifestSize` implementation.

Reply:
Yeah, that makes sense. I missed the RFC the first time around. Any `local` manifest size would be hypothetical anyways so `0` feels fine.